### DISCUSSION TITLE
How to change color of icons?

### DISCUSSION THREAD

**Discussion by Mr-Faisal-SGD** [ng-icons/ng-icons#60] [Q&A] [answered]
I am changing color like this
color="#FBB03B"
But this ain't working

**Comment by ashley-hunter** [comment]
Hi, try instead use CSS and the color property.

https://stackblitz.com/edit/angular-adpvfg?file=src%2Fmain.ts

**Comment by ashley-hunter** [comment]
The color input seems to work fine for me:

https://stackblitz.com/edit/angular-wjry4u?file=src%2Fmain.ts

if you are able to reproduce this not working in stackblitz please raise an issue.

**Comment by Mr-Faisal-SGD** [comment]
I don't know man, its working here but not in my project
![Screenshot_149](https://github.com/ng-icons/ng-icons/assets/100855617/06fb54dc-9b90-4476-87cb-503751f374d7)
![Screenshot_150](https://github.com/ng-icons/ng-icons/assets/100855617/1a09ce3b-3189-473b-b1b9-c2ebc99d75d4)

**Reply by ashley-hunter** [reply, marked as answer]
Sure, so I see the icon color is matching the text within the button which suggests the styles setting the color on the button has a higher specificity and is overriding the color.

An easy way to check would be to move the icon out of the button and see if it appears red. If not, it would suggest the color is being set more globally rather than by the button styles.

At worst a workaround would be to set the color on the icon using CSS and add !important, e.g.

<ng-icon style="color: red !important" ...>

**Reply by Mr-Faisal-SGD** [reply]
Thanks, it's working now
